### PR TITLE
Persist location on server

### PR DESCRIPTION
**Pull request by mauricecarrier7**
https://www.notion.so/lyrasis/iOS-Sync-audiobook-listening-position-to-server-98aaa6fa8dad4a60b617fafa5f830750?d=3a3ab32474014a79b8ecdd33bd7fd615

### CODE REVIEW

**Comment by mauricecarrier7** [comment]
Complete and ready for review

File: NYPLAudiobookToolkit/Core/AudiobookManager.swift [hunk 1]
```diff
@@ -194,7 +194,7 @@ extension DefaultAudiobookManager: PlayerDelegate {
         }
         if lastChapter.inSameChapter(other: chapter) {
             self.playbackCompletionHandler?()
-            self.audiobook.player.movePlayheadToLocation(firstChapter)
+            self.audiobook.player.movePlayheadToLocation(firstChapter, completion: nil)
```
Comment (vladimirfedorov) [inline NYPLAudiobookToolkit/Core/AudiobookManager.swift:197]: Do we still need this change? `completion` seems to be optional

File: NYPLAudiobookToolkit/Core/AudiobookTableOfContents.swift [hunk 2]
```diff
@@ -65,7 +65,7 @@ public final class AudiobookTableOfContents: NSObject {
 extension AudiobookTableOfContents: UITableViewDelegate {
     public func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let spineElement = self.networkService.spine[indexPath.row]
-        self.player.playAtLocation(spineElement.chapter)
+        self.player.playAtLocation(spineElement.chapter, completion: nil)
```
Comment (vladimirfedorov) [inline NYPLAudiobookToolkit/Core/AudiobookTableOfContents.swift:68]: Also here, `completion` seems to be optional

File: NYPLAudiobookToolkit/Core/AudiobookManager.swift (hunk 1 above)
Comment (mauricecarrier7) [reply NYPLAudiobookToolkit/Core/AudiobookManager.swift:197]: Since we are unable to set a default argument for a protocol, we have to provide a value.

**Review by vladimirfedorov — APPROVED** [review]
Looks good to me

**Pull request merged**